Source/WTF/wtf/cocoa/URLCocoa.mm

```diff
@@ -52,9 +52,8 @@ - (BOOL)_web_looksLikeIPAddress;
 
 RetainPtr<CFURLRef> URL::emptyCFURL()
 {
-    // We use the toll-free bridge to create an empty value that is distinct from null that no CFURL function can create.
```

Comment:
Was this comment incorrect? How exactly did you verify that this CFURLCreateWithString invocation is equivalent?

Reply:
Just to prove my point. I am going to do something. Watch this

Reply:
If the tests pass, then we know it is equivalent. Otherwise, the program will abort and all will crash down.